src/Relation/Binary/Construct/Constant.agda

```diff
@@ -8,13 +8,19 @@
 
 module Relation.Binary.Construct.Constant where
 
+open import Level
 open import Relation.Binary
 
+private
+  variable
+    a b c : Level
+    A : Set a
+    B : Set b
+
 ------------------------------------------------------------------------
--- Definition
+-- Re-export definition
 
-Const : ∀ {a b c} {A : Set a} {B : Set b} → Set c → REL A B c
-Const I = λ _ _ → I
+open import Relation.Binary.Construct.Constant.Core public
```

Comment:
Can't quite spot why do we need a `Core` module for this?

Reply:
`Setoid` induces a big dependency